docs/basic_usage.rst

```diff
@@ -54,12 +54,12 @@ AutocompletePanel
     to a model class or a model string in ``app_label.ModelName`` syntax.
 
     .. note::
-        Unlike :class:`~wagtail:wagtail.wagtailadmin.edit_handlers.PageChooserPanel`,
+        Unlike :class:`~wagtail:wagtail.admin.panels.FieldPanel`,
         ``AutocompletePanel`` does not support receiving ``target_model`` as a list.
 
     .. note::
         ``AutocompletePanel`` does not support receiving the ``can_choose_root``
-        argument that :class:`~wagtail:wagtail.wagtailadmin.edit_handlers.PageChooserPanel`
+        argument that :class:`~wagtail:wagtail.admin.panels.FieldPanel`
```

Comment:
You still need to use `PageChooserPanel` if you pass the `can_choose_root` kwarg

Reply:
This should be right as well but I thought it could do with a little more clarity injected.